### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by eksperimental**
Closes https://github.com/elixir-lang/ex_doc/issues/617

### CODE REVIEW

**Comment by josevalim** [comment]
It seems this commit removes the css file by accident?

File: LICENSE [hunk 1]
```diff
@@ -47,17 +47,17 @@ https://highlightjs.org/
...
+  Released under the MIT license.
+  https://github.com/jquery/jquery/blob/master/LICENSE.txt
 
 
 Includes Sizzle.js
-Copyright 2011, The Dojo Foundation
+Copyright jQuery Foundation and other contributors
 
-  Released under the MIT, BSD, and GPL Licenses.
+  Released under the MIT Licenses.
```
Comment (nscyclone) [inline LICENSE]: Shouldn't `Licenses` be in singular here?

**Comment by eksperimental** [comment]
all issues addressed

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**